CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# CHANGELOG
+
+## \[1.1.0\] - 2024-12-19
```

Comment:
```suggestion
## [1.1.0] - 2024-12-19
```

Reply:
Fixed in c0d60fa16116790c6e26299acfcaa3b560d71613.